extensions/positron-python/src/client/positron/session.ts

```diff
@@ -541,15 +547,15 @@ export class PythonRuntimeSession implements positron.LanguageRuntimeSession, vs
         this.adapterApi = ext?.exports as PositronSupervisorApi;
         const kernel = this.kernelSpec
             ? // We have a kernel spec, so we're creating a new session
-              await this.adapterApi.createSession(
```

Comment:
This changes are required by the linter even though they are not really part of this PR. Done in https://github.com/posit-dev/positron/pull/6400/commits/e12f88d94f8ce600d219bd06992f1b9150b20a9a